lightpath/path.py

```diff
@@ -234,13 +236,17 @@ def blocking_devices(self) -> List[Device]:
                 block.append(prev_device)
             # check inserted
             elif curr_state is DeviceState.Inserted:
-                # Ignore devices with low enough transmssion
+                current_transmission *= curr_status.transmission
+                # Blocking if single device has low enough transmssion
                 if curr_status.transmission < self.minimum_transmission:
                     block.append(device)
-            # Find unknown and inconsistent devices
+                # Any device seeing current transmission below min would block
+                elif current_transmission < self.minimum_transmission:
+                    block.append(device)
```

Comment:
Does this new total transmission condition make the old single device condition redundant?

Note that this is a change in behavior from previous- I think it's more useful this way

Reply:
I think this is also leftover from when I colored downstream half-removed devices as half-removed, not blocking.  It is redundant now.